### PR TITLE
UI fix for adding and editing child info.

### PR DESCRIPTION
**Pull request by laxmankhanal**

### CODE REVIEW

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 1]
```diff
@@ -218,4 +215,17 @@ class AddChildFormTVC: UITableViewController, UITextFieldDelegate {
     func hideKeyboard() {
         self.view.endEditing(true)
     }
+
+    override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let title = UILabel()
+        let headerFrame = self.tableView.frame
+        let headerView: UIView = UIView(frame: CGRectMake(0, 0, headerFrame.width, headerFrame.height))
+        print(headerFrame.width, headerFrame.height)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: remove print

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 2]
```diff
@@ -218,4 +215,17 @@ class AddChildFormTVC: UITableViewController, UITextFieldDelegate {
...
+        title.frame =  CGRectMake(8, 2, headerFrame.size.width-20, 24) //width equals to parent view with 8 left and right margin
+        title.text = self.tableView(tableView, titleForHeaderInSection: section) //This will take title of section from 'titleForHeaderInSection' method
+        headerView.backgroundColor = UIColor(red: 247/255, green: 247/255, blue: 247/255, alpha: 1)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: define the color in CVColor

File: Vaccination/ViewController/EditChildFormTVC.swift [hunk 3]
```diff
@@ -150,5 +150,16 @@ class EditChildFormTVC: UITableViewController {
         }
     }
 
+    override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let title = UILabel()
+        let headerFrame = self.tableView.frame
+        let headerView: UIView = UIView(frame: CGRectMake(0, 0, headerFrame.width, headerFrame.height))
+        print(headerFrame.width, headerFrame.height)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/EditChildFormTVC.swift]: remove prints

File: Vaccination/ViewController/EditChildFormTVC.swift [hunk 4]
```diff
@@ -150,5 +150,16 @@ class EditChildFormTVC: UITableViewController {
...
+        title.frame =  CGRectMake(8, 2, headerFrame.size.width-20, 24) //width equals to parent view with 8 left and right margin
+        title.text = self.tableView(tableView, titleForHeaderInSection: section) //This will take title of section from 'titleForHeaderInSection' method
+        headerView.backgroundColor = UIColor(red: 247/255, green: 247/255, blue: 247/255, alpha: 1)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/EditChildFormTVC.swift]: in CVColor

File: Vaccination/ViewController/AddChildFormTVC.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/AddChildFormTVC.swift]: :ok:

File: Vaccination/ViewController/AddChildFormTVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/AddChildFormTVC.swift]: :ok:

File: Vaccination/ViewController/EditChildFormTVC.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/EditChildFormTVC.swift]: :ok:

File: Vaccination/ViewController/EditChildFormTVC.swift (hunk 4 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/EditChildFormTVC.swift]: :ok:

**Comment by sanjaymhj** [comment]
:+1:

File: Vaccination/ViewController/EditChildFormTVC.swift [hunk 5]
```diff
@@ -150,5 +150,15 @@ class EditChildFormTVC: UITableViewController {
         }
     }
 
+    override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let title = UILabel()
+        let headerFrame = self.tableView.frame
+        let headerView: UIView = UIView(frame: CGRectMake(0, 0, headerFrame.width, headerFrame.height))
```
Comment (bishalg) [inline Vaccination/ViewController/EditChildFormTVC.swift]: we do no need to specify the UIView in this case Swift can auto infer

File: Vaccination/ViewController/EditChildFormTVC.swift [hunk 6]
```diff
@@ -150,5 +150,15 @@ class EditChildFormTVC: UITableViewController {
...
+        title.frame =  CGRectMake(8, 2, headerFrame.size.width-20, 24) //width equals to parent view with 8 left and right margin
```
Comment (bishalg) [inline Vaccination/ViewController/EditChildFormTVC.swift]: Uuse seift CGRect(x: ...) method rather then old CGRectMake

File: Vaccination/ViewController/EditChildFormTVC.swift [hunk 7]
```diff
@@ -150,5 +150,15 @@ class EditChildFormTVC: UITableViewController {
...
+        title.text = self.tableView(tableView, titleForHeaderInSection: section) //This will take title of section from 'titleForHeaderInSection' method
```
Comment (bishalg) [inline Vaccination/ViewController/EditChildFormTVC.swift]: write comment about the code one line above the code no in same line

File: Vaccination/ViewController/EditChildFormTVC.swift (hunk 5 above)
Comment (sanjaymhj) [inline Vaccination/ViewController/EditChildFormTVC.swift]: :ok:

File: Vaccination/ViewController/EditChildFormTVC.swift (hunk 6 above)
Comment (sanjaymhj) [inline Vaccination/ViewController/EditChildFormTVC.swift]: :ok:

File: Vaccination/ViewController/EditChildFormTVC.swift (hunk 7 above)
Comment (sanjaymhj) [inline Vaccination/ViewController/EditChildFormTVC.swift]: :ok:

**Pull request merged**